frontend/src/dep.ts

```diff
@@ -59,7 +60,15 @@ export function initUIFactory(
   const versionService = new VersionService(envService);
   const store = initStore();
   const searchService = new SearchService();
-  const changeLogService = new ChangeLogService();
+  const changeLogGraphQLApiService = new ChangeLogGraphQLApi(
+    authService,
+    envService,
+    graphQLService
+  );
```

Comment:
I saw you are creating a separate GraphQL API service for change log. Is this to prepare for spanning it to a new service? We have a generic GraphQL API client here: https://github.com/short-d/short/blob/master/frontend/src/service/ShortGraphQL.api.ts

Reply:
Resolved on slack, https://github.com/short-d/short/issues/770